src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonResult.cs

```diff
@@ -50,9 +50,7 @@ public JsonResult(object value, JsonSerializerSettings serializerSettings)
         /// </summary>
         public JsonSerializerSettings SerializerSettings { get; set; }
 
-        /// <summary>
-        /// Gets or sets the HTTP status code.
-        /// </summary>
+        /// <inheritdoc />
```

Comment:
`<inheritdoc />` really doesn't work. Might want to leave the doc if it's already there.

Reply:
Hmm. There's already [331 `<inheritdoc />` tags in the MVC codebase](https://github.com/aspnet/Mvc/search?q=inheritdoc). Are you sure they don't work?

Reply:
Unfortunately [not](https://github.com/aspnet/Mvc/issues/928) - the doc comments typically appear empty when you view them so it certainly helps to not replace existing docs with `<inhertidoc />`